app/scripts/controllers/transactions/index.test.js

```diff
@@ -772,6 +794,15 @@ describe('Transaction Controller', function () {
         .stub(txController, '_getDefaultGasFees')
         .callsFake(() => ({ gasPrice: TEST_GASPRICE }));
 
+      const stub3 = sinon
+        .stub(txController, 'getAdvancedGasFee')
+        .callsFake(() => ({
+          currentChainId: {
+            maxFeePerGas: TEST_MAX_FEE_PER_GAS,
+            maxPriorityFeePerGas: TEST_MAX_PRIORITY_FEE_PER_GAS,
+          },
+        }));
+
```

Comment:
Not sure why is thus stud needed now since it was required before.

Reply:
This PR was created long time ago...

Reply:
Are these still needed now ?

Reply:
I suppose so, as it can also be seen on the main branch, within the same file.

Reply:
`stub3` look new to this PR.

Reply:
Oh, I thought you were talking about the `maxPriorityFeePerGas: TEST_MAX_PRIORITY_FEE_PER_GAS`...
So, here is the thing. This PR introduces saving the `maxFeePerGas` and `maxPriorityFeePerGas` for every network separately. Because of it, those fees will look like this: 
`      "0x5": {
        "maxBaseFee": "75",
        "priorityFee": "2"
      }`
Are we on the same page now, I mean does this make sense to you or are you suggesting something else?